src/shared_modules/dbsync/src/dbsyncPipelineFactory.cpp

```diff
@@ -44,16 +44,30 @@ namespace DbSync
         }
         ~Pipeline()
         {
+            if (m_spSyncNode)
+            {
+                try
```

Comment:
It would be convenient to do 1 only try catch here and not 3?

Reply:
if we do just 1 try catch if some of the operations throw the others won't even be tried.